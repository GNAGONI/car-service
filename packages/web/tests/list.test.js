import React from 'react'
import renderer from 'react-test-renderer'
import List from 'components/list'

it('Renders correctly', () => {
  const tree = renderer.create(<List />).toJSON()
  expect(tree).toMatchSnapshot()
})
