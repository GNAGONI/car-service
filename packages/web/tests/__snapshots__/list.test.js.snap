// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<section
  className="list"
>
  <div
    className="list__row list__row-header"
  />
</section>
`;
